docker-for-mac/networking.md

```diff
@@ -39,9 +39,30 @@ $ docker run -p 8000:80 -d nginx
 Now, connections to `localhost:8000` are sent to port 80 in the container. The
 syntax for `-p` is `HOST_PORT:CLIENT_PORT`.
 
+### SSH agent forwarding
```

Comment:
Should this topic be in either "file system sharing" (https://docs.docker.com/docker-for-mac/osxfs/) or a separate page? (it's not directly related to `networking`)

Reply:
Yes!

Reply:
I've moved the content to the file system sharing topic as suggested.